### DISCUSSION TITLE
Introduce yourself

### DISCUSSION THREAD

**Discussion by 0xWDG** [AuroraEditor/AuroraEditor#74] [General]
👋,

In this discussion you can introduce yourself

---

Hi 👋, I'm Wesley de Groot.
I do software (backend, frontend) & hardware (blood group measuring robots).
I write Swift, PHP, HTML, JavaScript, CSS and more.

if you have any questions please ask them 😊

**Comment by Maartz** [comment]
Hi,

I'm William, I'm a senior software engineer specialised in backend.
I worked with Ruby, Ruby on Rails, Golang on Uber-like application. Around 20 µS.
I also used to work at Thales on industrial software and warehouse managment.
Right now, I work with Elixir, Phoenix with CQRS. And a few Rust lambdas.

Cheers!
